event_sampler/resources.py

```diff
@@ -68,16 +70,20 @@ def get_histogram(self):
                 'bin_cumulative_sum': numpy.cumsum(ar).tolist()}
 
     def get_status(self):
-        last_event = self.sampler.last_event()
         web3 = self.sampler.chain.web3
         ret = {}
         ret['auction_stage'] = self.contract.call().stage()
         ret['price'] = self.contract.call().price()
         block_to_sum = {}
         for block, events in self.sampler.events.items():
             block_to_sum[block] = sum((event['args']['_amount'] for event in events))
+        total = sum((v for v in block_to_sum.values()))
+        wallet_balance = self.contract.web3.eth.getBalance(self.wallet_address)
+        if total != wallet_balance:
```

Comment:
so if they don't match you simply just take the value of the wallet balance. Will this affect the graph? Also why not just take the wallet balance to begin with?

Reply:
that's just a check to raise a warning if for some reason stored events sum doesn't match real wallet balance. In the result returned from the endpoint actual balance is always used.